packages/admin/src/lib/framework-utils.ts

```diff
@@ -2,20 +2,36 @@ import { Mastra } from '@mastra/core';
 import fs from 'fs';
 import path from 'path';
 
-function getFramework() {
+import fse from 'fs-extra/esm';
+
+async function loadConfig() {
+  const configPath = `${process.env.CONFIG_PATH}.ts`;
+  if (!process.env.CONFIG_PATH) {
+    return undefined;
+  }
+
+  const configFileString = fs.readFileSync(configPath, 'utf8');
+
+  const outputFilePath = path.join(process.cwd(), 'temp-mastra-config.ts');
+
+  await fse.outputFile(outputFilePath, configFileString);
+}
+
+async function getFramework() {
   try {
-    const { config } = require(process.env.CONFIG_PATH!);
+    await loadConfig();
 
-    const framework = Mastra.init(config);
+    const { config } = require('../../temp-mastra-config.ts');
```

Comment:
Interesting. I'm guessing this has something to do with how compilation/bundling happens? If we hardcode the path then it works correctly, but we need to dynamically load the config in a different way (by reading the file and writing it to a static location)?

Reply:
yeah
if i run require on the CONFIG_PATH directly, it fails with this error ` server relative imports are not implemented yet. Please try an import relative to the file you are importing from.`
so what i do is first read the file and write to a new temp file, and then run require on that new temp file path. and the temp file is git ignored, so it's never pushed.